jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/utils/Users.groovy

```diff
@@ -62,18 +62,18 @@ abstract class Users {
     private EncryptionService encryptionService
     MockMvc mockMvc
 
-    def PermissionGranter create() {
+    def PermissionGranter create(User user) {
         def group = groupDao.getGroupByName(AdministrationGroup.USER.name)
-        def user = userDao.getByUsername(USERNAME)
-        if (user == null) {
-            user = new JCUser(USERNAME, 'sample@example.com', encryptionService.encryptPassword(PASSWORD))
-            user.enabled = true
-            user.addGroup(group)
-            userDao.saveOrUpdate(user)
+        def fromDb = userDao.getByUsername(user.username)
+        if (fromDb == null) {
+            fromDb = new JCUser(user.username, 'sample@example.com', encryptionService.encryptPassword(user.password))
+            fromDb.enabled = true
+            fromDb.addGroup(group)
+            userDao.saveOrUpdate(fromDb)
             userDao.flush()
         }
         //Needed for managing permissions
-        setAuthentication(new JCUser(USERNAME, 'sample@example.com', PASSWORD));
+        setAuthentication(new JCUser(user.username, 'sample@example.com', user.password));
```

Comment:
Because to `fromDb` has encrypted password. If we try to reuse it we'll get authentication error.